v0/customerio/transform.js

```diff
@@ -158,40 +147,31 @@ function responseBuilder(message, evType, evName, destination) {
     }
   }
   const payload = removeUndefinedValues(rawPayload);
-  //console.log(payload);
+  // console.log(payload);
   response.endpoint = endpoint;
   response.method = requestConfig.requestMethod;
-  response.headers = {
-    "Content-Type": "application/json",
-    Authorization:
-      "Basic " +
-      btoa(destination.Config.siteID + ":" + destination.Config.apiKey)
-  };
   response.body.JSON = payload;
 
-  //console.log(response);
+  // console.log(response);
   return response;
 }
 
 function processSingleMessage(message, destination) {
-  let messageType = message.type.toLowerCase();
+  const messageType = message.type.toLowerCase();
   let evType;
   let evName;
-  //console.log("messageType", messageType);
+  // console.log("messageType", messageType);
   switch (messageType) {
     case EventType.IDENTIFY:
       evType = "identify";
       break;
     case EventType.PAGE:
       evType = "page"; // customerio mandates sending 'page' for pageview events
-      evName = message.name ? message.name : message.properties.url;
+      evName = message.name || message.properties.url;
       break;
     case EventType.SCREEN:
       evType = "event";
-      evName =
-        "Viewed " + message.name
-          ? message.name
-          : message.properties.name + " Screen";
+      evName = "Viewed " + message.name || message.properties.name + " Screen";
```

Comment:
"Viewed " + (message.name || message.properties.name) + " Screen" 
For better readability?